evm/chains/base.py

```diff
@@ -310,6 +301,12 @@ def get_chaindb_class(cls) -> Type[BaseChainDB]:
     def get_vm_configuration(cls) -> Tuple[Tuple[int, Type['BaseVM']], ...]:
         return cls.vm_configuration
 
+    def default_header(self, header: BlockHeader=None) -> BlockHeader:
```

Comment:
Having a method with this name that takes an argument feels odd to me. Even more so since it simply returns the argument passed to it when it's not None. Is this really needed?